cmd/snap-repair/runner.go

```diff
@@ -197,21 +197,22 @@ func readStatus(r io.Reader) RepairStatus {
 
 // errtrackerReport reports an repairErr with the given logPath to the
 // snap error tracker.
-func (r *Repair) errtrackerReport(repairErr error, logPath string) error {
+func (r *Repair) errtrackerReport(repairErr error, status RepairStatus, logPath string) error {
 	errMsg := fmt.Sprintf("%s", repairErr)
 
 	scriptOutput, err := ioutil.ReadFile(logPath)
 	if err != nil {
 		logger.Noticef("cannot read %s", logPath)
 	}
-	dupSig := fmt.Sprintf("%s\noutput:\n%s", errMsg, scriptOutput)
+	dupSig := fmt.Sprintf("%s/%s\n%s\noutput:\n%s", r.BrandID(), r.RepairID(), errMsg, scriptOutput)
 
 	extra := map[string]string{
 		"Revision": strconv.Itoa(r.Revision()),
 		"BrandID":  r.BrandID(),
 		"RepairID": r.RepairID(),
+		"Status":   status.String(),
 	}
-	_, err = errtrackerReportRepair(r.Ref().String(), errMsg, dupSig, extra)
+	_, err = errtrackerReportRepair(fmt.Sprintf("%s/%s", r.BrandID(), r.RepairID()), errMsg, dupSig, extra)
```

Comment:
could maybe compute the label %s/%s once? at the start

Reply:
Thanks for this suggestion, I am doing this now. (fun fact: I was considering this while writing the code but couldn't think of a good name for the variable - but it does make a lot of sense)